querybook/webapp/components/DataTableViewOverview/DataTableViewOverview.tsx

```diff
@@ -82,6 +82,24 @@ function useRefreshMetastore(table: IDataTable) {
     return [handleRefreshTable, isRefreshing] as const;
 }
 
+function createChannelLinkDOM(customProperties) {
+    const channelsKey = 'channels';
+    const channelsValue = customProperties[channelsKey]?.toString() ?? '';
+    const channelsName = channelsValue.slice(1);
+    const channelLink = `https://pinterest.slack.com/app_redirect?channel=${channelsName}`;
```

Comment:
this is too Pinterest specific for the opensource repo and need a better solution for it. let's not add a link for now. maybe can add link in the properties when syncing from pincat

Reply:
removed link for now